locales/fr-FR.json

```diff
@@ -10,7 +10,8 @@
     "message": "n'est pas vide. Supprimer les fichiers existants et continuer\u00a0?"
   },
   "packageName": {
-    "message": "Nom du package\u00a0:"
+    "message": "Nom du package\u00a0:",
+    "invalidMessage": "Invalide package.json nom du"
```

Comment:
In French, that would be `Le nom du package.json est invalide`

Reply:
Thank you for your review, it has been revised.